bazel-mode.el

```diff
@@ -26,13 +26,21 @@
   :link '(url-link "https://github.com/bazelbuild/emacs-bazel-mode")
   :group 'languages)
 
-(defcustom bazel-mode-buildifier-cmd "buildifier"
+(defcustom bazel-mode-buildifier-command "buildifier"
```

Comment:
I'd do this rename in a separate commit since it's quite unrelated

Reply:
Done.